DESCRIPTION

```diff
@@ -15,6 +15,7 @@ Maintainer: Carsten Urbach <urbach@hiskp.uni-bonn.de>
 SystemRequirements: Gnu Scientific Library version >= 1.8
 Description: Toolkit to extract hadronic quantities from Lattice QCD simulations. It contains functionality for IO, plotting, bootstrap and jackknife resampling, fitting, GEVP solving, error and autocorrelation estimation as well as other areas.
 Imports: boot, abind
+Depends: boot, R6
```

Comment:
I guess this should now just be in "Imports", right?

Reply:
I had to read [this](http://r-pkgs.had.co.nz/description.html) yet another time, and I am not certain that I got all the details right. But I think that having it in `Imports` is the right thing, there is no reason to use `Depends` for us.

The `R6` is somewhat special as we need that during compilation time. From [this footnote](https://adv-r.hadley.nz/r6.html#fn51) it seems that `Imports` is just fine.